1-js/02-first-steps/06-alert-prompt-confirm/article.md

```diff
@@ -12,7 +12,7 @@ Por exemplo:
 alert('Olá');
 ```
 
-A mini-janela com a mensagem é chamada de _modal window_. A palavra "modal" significa que o visitante não pode interagir com o resto da página, pressionar outros botões, etc, até que ele tenha lidado com a janela. Nesse caso -- até pressionar "OK".
+A mini-janela com a mensagem é chamada de _modal window_. A palavra "modal" significa que o visitante não pode interagir com o resto da página, pressionar outros botões, etc., até ele ter lidado com a janela. Nesse caso -- até pressionar "OK".
 
 ## prompt
 
```

Comment:
```suggestion
O visitante pode digitar algo no campo de entrada do prompt e pressionar OK. Então nós temos esse texto no `result`. Ou ele pode cancelar a entrada pressionando Cancelar ou `key:Esc`, então nós temos `null` como `result`.
```

Reply:
Done in https://github.com/javascript-tutorial/pt.javascript.info/pull/244/commits/e072d452a5d1a24871c5bf9e1fba0ff81ad4a6a5